pre_pr.sh

```diff
@@ -5,7 +5,7 @@ set -e
 set -x
 
 flutter format lib
-flutter format test
+# flutter format test
```

Comment:
why no need to format test files?

Reply:
there are no `test` file in the first place 🙈 .
If tests are added, uncomment this line.

Reply:
okok